docs/codecs/known-issues.md

```diff
@@ -39,9 +35,8 @@ title:  "Warnings and known issues"
    range of test cases, and is likely to _just work_ for you, but it relies on macros (provided by
    Shapeless) that rely on compiler functionality that is not always perfectly robust
    ("[SI-7046](https://issues.scala-lang.org/browse/SI-7046) is like [playing roulette](https://twitter.com/li_haoyi/status/637281580847878145)"), and if you're running into
-   problems, it's likely that they're not your fault. Please file an issue here or ask a question on
-   the [Gitter channel](https://gitter.im/circe/circe), and we'll do our best to figure out whether the problem is
-   something we can fix.
+   problems, it's likely that they're not your fault. Please file an issue here or ask a question on [Discord](https://discord.gg/XF3CXcMzqD), 
+   and we'll do our best to figure out whether the problem is something we can fix.
```

Comment:
Can we link to, or reference the specific channel within the typelevel discord?

Reply:
I'll give it a shot, sure.